### ISSUE TITLE
[Sourcing][Accessibility][ATL Testing][Multi-input token][more input][P4]This SPAN has an aria-setsize attribute but does not have an aria-posinset attribute

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10383]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
This SPAN has an aria-setsize attribute but does not have an aria-posinset attribute
Note: This issue is common for all the selected list items

<img width="451" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/88182358/ba0cf411-5cbe-4bd1-9785-986b13f63c29">

Expected:
When ARIA attributes (state, roles, and properties) are not used correctly assistive technology may not correctly function as expected. Valid markup should always be used.

#### If this is a bug, please provide steps for reproducing it.
Tested on https://sap.github.io/fundamental-ngx/#/core/multi-input#simple

#### Please provide relevant source code if applicable.
Code Snippet:
<span aria-setsize="1" aria-readonly="false" aria-selected="false" class="fd-token" tabindex="-1" role="option"><span class="fd-token__text no-text-select"><span>Astro Laptop 1516</span></span><span aria-label="Deletable" class="fd-token__close" role="button"></span></span>

**Comment by droshev** [comment]
@I543348 We would need more information and steps how to reach to the issue

**Comment by I543348** [comment]
Hi @droshev , This is reported from ATL testing ,
Could you please check with ATL team?
Thanks

**Comment by fkolar** [comment]
@I543348, can you update the Expected part? This is copied from some site but what exactly needs to be done here? 
Either remove areasetsize or it must be always used with combination with this other area tag? What do you expect these tags will reflect? what value ?

**Comment by I543348** [comment]
Hi @fkolar , This was reported by ATL team and i have asked them to answer few queries on this.
Thanks.

**Comment by nikolay-kolarov** [comment]
Hi Colleagues,

I also noticed this in the numbering of the tokens in MultiInput. Aria-posinset and aria-setsize go together.

Here is their spec in ARIA: https://www.w3.org/TR/wai-aria-1.2/#aria-posinset and https://www.w3.org/TR/wai-aria-1.2/#aria-setsize.

As example I can give the following:

`<fd-token aria-posinset="1" aria-setsize="3" class="ng-star-inserted" style="max-width: 100%; display: inline-block; visibility: visible;">
<span role="option" tabindex="-1" class="fd-token" aria-roledescription="token" aria-selected="false" aria-readonly="false" aria-setsize="3"><span class="fd-token__text no-text-select"><span class="ng-star-inserted">Delaware</span></span><span role="button" class="fd-token__close ng-star-inserted" aria-label="Deletable"></span></span></fd-token>`

The correct way to go forward is to remove the posinset and setsize from fd-token and put them together on the span with role option:
`<fd-token class="ng-star-inserted" style="max-width: 100%; display: inline-block; visibility: visible;"><span role="option" tabindex="-1" class="fd-token" aria-roledescription="token" aria-selected="false"  aria-posinset="1" aria-setsize="3"><span class="fd-token__text no-text-select"><span class="ng-star-inserted">Delaware</span><!----></span><span role="button" class="fd-token__close ng-star-inserted" aria-label="Deletable"></span><!----></span><!----></fd-token>`

Best Regards,
Nikolay

**Comment by sagar7259** [comment]
Hello,

This issue is false positive, has not this issue is no more to be found or detected by AMP tool. Hence this could be closed.

Thanks,
Sagar (ATL team)

**Comment by nikolay-kolarov** [comment]
Hi Colleagues,

In this case, I would advise you to keep this then on the backlog and maybe not treat it as a bug, but as a future enhancement, as the rendering can be optimized.

Best Regards,
Nikolay
